lib/commands/general.js

```diff
@@ -93,5 +93,16 @@ commands.launchApp = iosCommands.general.launchApp;
 
 commands.closeApp = iosCommands.general.closeApp;
 
+commands.keys = async function (keys) {
+  if (!this.isWebContext()) {
+    throw new errors.UnknownError('Command should be proxied to WDA');
+  }
+  let el = await this.active();
```

Comment:
I got it from here https://github.com/appium/appium-ios-driver/blob/master/lib/commands/general.js#L113-L131
Thats how we have been doing it :)
I guess there is no keyboard sendkeys in `webview` world. Just guess!